example/schema.avsc

```diff
@@ -708,7 +708,7 @@
                 }
               },
               {
-                "name": "value",
+                "name": "ivalue",
```

Comment:
A couple of things here:
1. You've changed the example schema here, but haven't regenerated the example codegen - we need to keep these in sync.
2. It seems like the change to using `ivalue` and `svalue`, to sidestep the field names colliding, deliberately hides the problem you mention in https://github.com/hchasestevens/avro2py/issues/16 - that seems like a bad solution. I would instead much prefer that you alter `requirements.txt` to specify `avro<={MAX COMPATIBLE VERSION}` so that we are explicit about which versions we're _actually_ compatible with.

Reply:
Huh I didn't notice that there was a generated section checked in. If this stayed changed I could regenerate that.

I'm not sure I agree with the pitch that we're incompatible with the new version of avro, there's nothing special we're doing that is broken, it's the new version of avro that is broken for anyone who writes using the datum writer. We just happen to have a test that exposes that failure due to it's very generically named fields of `id` and `value`.

I don't think this is a bad solution any more than pinning the dependency into the past to avoid breaking a test is, but if you really do strongly prefer it I can do it. I checked and it seems like the bigger changes we're missing out on are:
 - support for UUIDs
 - allowing namespaces named `""`
 - validation is no longer implemented recursively and provides better error messages (this is the change that breaks this test)
 - fixing some imports that were causing conflicts with other dependencies

So that's the tradeoff, but nothing there sounds critical.

Reply:
I pinned the version and reverted the change.